analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -26,7 +26,7 @@
 
   <ItemGroup>
     <PackageReference Include="altcover" Version="8.6.14" />
-    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="FluentAssertions" Version="6.8.0" />
```

Comment:
6.10.0 is now available - #6830

Reply:
I looked into the change log and there were quite some changes from 6.8. to 6.10. I'd like to stay and 6.8. for this PR and dot the minor upgrade separately.